### PR TITLE
Remove session jwt from header

### PR DESCRIPTION
**Pull request by aviadl**
Remove auto refresh in case there is no session jwt Unless there is a refresh
related to https://github.com/descope/etc/issues/1095

### CODE REVIEW

File: descope/auth/auth.go [hunk 1]
```diff
@@ -343,15 +343,15 @@ func (auth *authenticationService) validateSession(sessionToken string, refreshT
 			logger.LogError("cannot validate refresh token, refresh expiration will not be available", tErr)
 		}
 	}
-	if sessionToken == "" || err != nil || forceRefresh {
+	if err != nil || forceRefresh {
```
Comment (slavikm) [inline descope/auth/auth.go:346]: Why did you remove? If no sessionToken provided, shouldn't we refresh and return it?

File: descope/descope.go [hunk 2]
```diff
@@ -33,8 +33,8 @@ type Config struct {
 	LogLevel logger.LogLevel
 	// LoggerInterface (optional, log.Default()) - set the logger instance to use for logging with the sdk.
 	Logger logger.LoggerInterface
-	// State whether session jwt should be sent to client in header or in the cookie,
-	// defaults to header, use cookie if session jwt will stay small (less than 1k)
+	// State whether session jwt should be sent to client in the cookie, or let the calling function handling the transfer of it
```
Comment (slavikm) [inline descope/descope.go:36]: `or let the calling function handle the transfer of the jwt`

File: descope/auth/auth.go (hunk 1 above)
Comment (aviadl) [reply descope/auth/auth.go:346]: The idea is that there is no auto refresh 
I should improve the condition
Need to add the config lookup on session via cookie attribute
